### DISCUSSION TITLE
add_packages add_requires 有何不同

### DISCUSSION THREAD

**Discussion by Smalldy** [xmake-io/xmake#2424] [General]
add_requires 既可以添加 包也可以添加其他target
add_package 只能添加包
是这样的吗？

**Comment by waruqi** [comment]
add_requires 全局接口，仅用于配置和安装包，不跟 target 绑定

add_packages 将 add_requires 引入的包，对特定 target 绑定 links includes 。。

具体看文档，都有详细描述
